.github/pull_request_template.md

```diff
@@ -1,15 +1,5 @@
 ## Checklist
 
- - [ ] The **PR Title** explains the impact of the change.
- - [ ] The **PR description** provides context as to why the change should occur and what the code contributes to that effect. This could also be a link to a JIRA ticket or a Github issue, if there is one.
- - [ ] If this PR changes development workflow or dependencies, they have been A) automated and B) documented under docs/developer/. All efforts have been taken to minimize development setup breakage or slowdown for co-workers.
- - [ ] If HTTP endpoint paths have been added or renamed, or feature configs have changed, the **endpoint / config-flag checklist** (see Wire-employee only backend [wiki page](https://github.com/zinfra/backend-wiki/wiki/Checklists)) has been followed.
- - [ ] If a cassandra schema migration has been added, I ran **`make git-add-cassandra-schema`** to update the cassandra schema documentation.
- - [ ] **changelog.d** contains the following bits of information ([details](https://github.com/wireapp/wire-server/blob/develop/docs/developer/changelog.md)):
-   - [ ] A file with the changelog entry in one or more suitable sub-sections. The sub-sections are marked by directories inside `changelog.d`.
-   - [ ] If new config options introduced: added usage description under docs/reference/config-options.md
-   - [ ] If new config options introduced: recommended measures to be taken by on-premise instance operators.
-   - [ ] If a cassandra schema migration is backwards incompatible (see also [these docs](https://github.com/wireapp/wire-server/blob/develop/docs/developer/cassandra-interaction.md#cassandra-schema-migrations)), measures to be taken by on-premise instance operators are explained.
-   - [ ] If a data migration (not schema migration) introduced: measures to be taken by on-premise instance operators.
-   - [ ] If public end-points have been changed or added: does nginz need un upgrade?
-   - [ ] If internal end-points have been added or changed: which services have to be deployed in a specific order?
+ - [ ] Add a new entry in a appropriate subdirectory of `changelog.d`
```

Comment:
this is also covered by the next point, but i guess it's good for me that i'll have it here because i forget every single time, still.  :)

Reply:
True, but we decided to add it anyway because it's needed by essentially all PRs. The idea is that the PR author would take relevant checklists from the guidelines document and add them to the PR description, and this one item will basically always be there anyway.